techniques/system/common/1.0/update.st

```diff
@@ -108,6 +108,8 @@ bundle common server_info
       "policy_server" string => ifelse(fileexists("${policy_server_file}"),
                                   readfile("${policy_server_file}", 2048),
                                   "");
+      "cfserved"      string => "${policy_server}";
```

Comment:
a comment on the reason of this variable and that it should be kept would have been welcome